### DISCUSSION TITLE
Has anyone worked on dockerizing this yet?

### DISCUSSION THREAD

**Discussion by grahamIT** [openrport/openrport#39] [Q&A]
I want to create a docker container so it can be easily deployed with a docker compose.

**Comment by alhdo** [comment]
There are dockerized project from the previous project that are closed source now. You can start with them to create a docker container. Would be very useful.

**Comment by IOTech17** [comment]
@grahamIT  @alhdo 

Please find a dockerized version of OpenRport : https://github.com/IOTech17/openrport-docker

**Reply by alhdo** [reply]
That will be usefull for the front-end dev. Thank you
